lib/fika/parser/literal_exps.ex

```diff
@@ -79,6 +79,30 @@ defmodule Fika.Parser.LiteralExps do
     |> label("record")
     |> Helper.to_ast(:record)
 
+  map_key_value =
+    allow_space
+    |> concat(exp)
```

Comment:
@emilsoman regarding what we allow for keys and as you mentioned on discord that we don't have "comparable" type.
am not entirely sure about allowing the whole of exp. Should we keep a subset of exp, for eg string,int,float ... or is this fine for now?

Reply:
Allowing exp is fine